lib/algos/set/ArrayUtils.js

```diff
@@ -0,0 +1,87 @@
+function indexOf(arr, value) {
```

Comment:
might be worth unit testing this function thoroughly (correctness, and in case it has corner cases causing infinite loops etc.) if not completely correct it could be a source of nasty bugs later on